catalog/spatial/registry/registry-admin-modules/registry-admin-remote-ui/src/main/webapp/main.js

```diff
@@ -95,7 +95,7 @@
         'marionette',
         'icanhaz',
         'js/application',
-        '../../../admin/js/HandlebarsHelpers',
+        'js/HandlebarsHelpers',
```

Comment:
Is `js/HandlebarsHelpers` a new copy of what was under the `admin` module? If so, isn't there a way we could share the files or extend from it somehow? If not, do we _really_ need everything that's in that file?